universite-catholique-de-louvain-histoire.csl

```diff
@@ -0,0 +1,388 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize="false" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Université Catholique de Louvain - Histoire (French)</title>
+    <title-short>UCL</title-short>
+    <id>http://www.zotero.org/styles/universite-catholique-de-louvain-histoire</id>
+    <link href="http://www.zotero.org/styles/universite-catholique-de-louvain-histoire" rel="self"/>
+    <link href="https://wordpress.com/post/lesvoyagesdebalnibarbi.wordpress.com/16" rel="documentation"/>
```

Comment:
@pbieswal, is there no official documentation we can link to? (even if it's e.g. just to a catalogue entry of a printed style guide)

Your link also asks me to log in. Did you mean to link to https://lesvoyagesdebalnibarbi.wordpress.com/2018/02/17/zotero-csl-ucl/ ?